resources/windows/wix/powershell.wxs.erb

```diff
@@ -23,6 +23,16 @@
           KeyPath="yes" />
       </Component>
 
+      <Component
+        Id="about_bolt_debugging.help.txt"
+        Directory="PowerShellBoltModuleHelpDir"
+        Guid="90b31dd1-28ef-4276-b672-8e87d432f70f">
```

Comment:
All of these new guides need their own GUID. You can generate those in PowerShell with the `New-Guid` cmdlet.

Reply:
Ah, some of the existing ids weren't unique - updated those too!